bastet_js/src/JsArray.re

```diff
@@ -93,7 +89,7 @@ module Unfoldable: UNFOLDABLE with type t('a) = array('a) = {
 
   let rec unfold = (f, init) =>
     switch (f(init)) {
-    | Some((a, next)) => Js.Array.concat([|a|], unfold(f, next))
+    | Some((a, next)) => Alt.alt([|a|], unfold(f, next))
```

Comment:
Could you do the same for `bastet/src/Array.re` and have it reference `Alt.alt`?